test/runnable/cppa.d

```diff
@@ -605,13 +605,19 @@ extern(C++)
 
 version (CRuntime_Microsoft)
 {
-    struct __c_long_double
+    version (PULL8152)
```

Comment:
What's the reasoning for the version statement? Will this test be run on a compiler that doesn't contain this change?

Reply:
Can't have two global declarations of __c_long, etc.